webpack.playground.config.js

```diff
@@ -138,11 +139,13 @@ module.exports = {
     }],
     // https://webpack.js.org/configuration/dev-server/#devserversetupmiddlewares
     setupMiddlewares(middlewares) {
-      const script = path.resolve(__dirname, 'playground/mocks/server.js');
-      const watch = [path.resolve(__dirname, 'playground/mocks')];
-      const env = { MOCK_SERVER_PORT, DEV_SERVER_PORT, BASE_URL: require(WIDGET_RC_JS).baseUrl };
-      nodemon({ script, watch, env, delay: 50 })
-        .on('crash', console.error);
+      if (!DISABLE_MOCK_SERVER) {
```

Comment:
nit: prefer positive logic when possible:
```suggestion
      if (DISABLE_MOCK_SERVER) {
        return middlewares;
      }
```